### PR TITLE
Quality's test cases for Desktop

### PR DESCRIPTION
**Pull request by tierninho**
manifest for Quality to execute test cases against Desktop builds (WIP)

### CODE REVIEW

File: docs/process/testing.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
+### Download Desktop
+  - [ ] User can download latest (Mac & Windows) Desktop from https://desktop.github.com/    
+    - [ ] Mac: https://central.github.com/mac/latest 
+      - [ ] Windows: https://github-windows.s3.amazonaws.com/GitHubSetup.exe
```
Comment (j-f1) [inline docs/process/testing.md]: Dedent this line?

File: docs/process/testing.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+  - [ ] Release notes page is up-to-date
+  - [ ] Help page is accessible
+
+### Welcome Flow
+  - [ ] Create your account (/join?source=github-desktop)
+    - [ ] User is not automatically logged into Desktop post creation
+  - [ ] Sign in to Github.com
```
Comment (j-f1) [inline docs/process/testing.md]: `GitHub.com`, maybe link to it?

File: docs/process/testing.md [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+    - [ ] `Cancel` returns to initial Welcome Flow
+    - [ ] `Sign in using your browser` opens default browser
+  		- [ ] Browser login, Authorize GitHub Desktop, “accept” link
+  - [ ] Sign in to Enterprise (v2.8 and up)
+    - [ ] `Continue` successful if server address is valid, else error message
+  		- [ ] `Sign in using your browser` opens default browser
+  - [ ] Browser login, [insert custom security measure], Authorize GitHub Desktop, “accept” link
+    - [ ] User goes to Configure Git if successful
+  - [ ] `Cancel` returns to initial Welcome Flow
+    - [ ] `Cancel` returns to initial Welcome Flow
```
Comment (j-f1) [inline docs/process/testing.md]: Is this a duplicate?

File: docs/process/testing.md [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+			  - [ ] Error message if code is wrong or inactive 
+  		- [ ] Error message if incorrect username/email or password
+    - [ ] Forgot link (/password_reset)
+    - [ ] `Cancel` returns to initial Welcome Flow
+    - [ ] `Sign in using your browser` opens default browser
+  		- [ ] Browser login, Authorize GitHub Desktop, “accept” link
+  - [ ] Sign in to Enterprise (v2.8 and up)
+    - [ ] `Continue` successful if server address is valid, else error message
+  		- [ ] `Sign in using your browser` opens default browser
+  - [ ] Browser login, [insert custom security measure], Authorize GitHub Desktop, “accept” link
```
Comment (j-f1) [inline docs/process/testing.md]: Indent this block?

File: docs/process/testing.md [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+    - [ ] `Delete` button deletes branch name
+    - [ ] `Cancel` button closes modal
+  - [ ] Update from Default Branch (cannot be master)
+  - [ ] Merge into Current Branch
+    - [ ] Use can filter existing branches
+    - [ ] User can select branch, other than current
+    - [ ] `Merge` button only activated if something to merge
+    - [ ] `Cancel` button closes modal
+  - [ ] Compare on GitHub (Mac: ⇧⌘C; Windows: Ctrl+Shift+C) (if repo already published on .com)
+  - [ ] Create Pull request (Mac: ⇧⌘P; Windows: Ctrl+Shift+P -- both cmds to change) opens Pull Request on .com
```
Comment (tierninho) [inline docs/process/testing.md]: Swap to Cmd-R

File: docs/process/testing.md (hunk 2 above)
Comment (tierninho) [inline docs/process/testing.md]: clarified it is a link, but no hyperlink to dotcom as that is not part of welcome flow

**Comment by tierninho** [comment]
@j-f1 yes, spaces is better as the first commit revealed the inconsistent indenting... should be fixed now

File: docs/process/testing.md [hunk 6]
```diff
@@ -0,0 +1,232 @@
...
+  - [ ] Drag and drop repository
+    - [ ] User can drag existing local repository into Desktop
+      - [ ] Successful attempt adds repo to Repo list; else error message
+
+### Application
+  - [ ] Minimize, Maximize, Close buttons work in top nav
+  - [ ] Double-clicking local desktop icon opens the application
+  - [ ] Double clicking top nav bar toggles full-screen / last used screen-size
+
+### GitHub Desktop menu
```
Comment (shiftkey) [inline docs/process/testing.md]: Maybe call out this is macOS only? Or maybe reorganize this based around the features themselves?

On Windows:
 - the `About` top-level menu is home to the `About GitHub Desktop` functionality
 - the `File` top-level menu is home to the `Options` functionality (different name) 
 - no options for installing command line tool (we do this automagically)
 - `File` | `Exit`

File: docs/process/testing.md [hunk 7]
```diff
@@ -0,0 +1,232 @@
+### Download Desktop
+  - [ ] User can download latest (Mac & Windows) Desktop from https://desktop.github.com/    
+    - [ ] Mac: https://central.github.com/mac/latest
```
Comment (shiftkey) [inline docs/process/testing.md]: These are the installer URLs for the classic apps. I'd steal [the URLs from here](https://github.com/desktop/desktop#where-can-i-get-it) for the Electron apps.

File: docs/process/testing.md (hunk 6 above)
Comment (tierninho) [inline docs/process/testing.md]: thanks, made most these changes and/or clarified the existing content 👍

File: docs/process/testing.md [hunk 8]
```diff
@@ -0,0 +1,366 @@
...
+  - [ ] When a branch is local-only make sure that the `Fetch` button changes to `Publish` and it publishes
+  
+### Publishing only
+  - [ ] Unpublished repository, unborn HEAD - `Publish button` enabled (user can publish repository)
+  - [ ] Unpublished repository, valid branch - `Publish button` enabled (user can publish repository and branch)
+  - [ ] Published repository, unborn HEAD - `Publish button` disabled (no branch to push)
+  - [ ] Published repository, branch without tracking - `Publish button` enabled (user can publish branch)
+  - [ ] Published repository, network action - `Publish button` disabled (don't interfere with existing action)
+  
+### Dotcom
```
Comment (shiftkey) [inline docs/process/testing.md]: Nit: "Dotcom" is [slang](https://en.wikipedia.org/wiki/Slang) - maybe we could say "GitHub The Website" here for those who aren't up on the lingo?

**Pull request merged**